ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh

```diff
@@ -52,6 +52,7 @@ v411="baremetal marketplace openshift-samples"
 # shellcheck disable=SC2034
 v412=" ${v411} Console Insights Storage CSISnapshot"
 v413=" ${v412} NodeTuning"
+v414=" ${v413} MachineAPI"
 latest_defined="v413"
```

Comment:
I guess we also need to update this line to `v414` ?

Reply:
good catch, fixed.